packages/ketcher-core/src/application/render/restruct/rerasterImage.ts

```diff
@@ -0,0 +1,53 @@
+import { ReObject, ReStruct } from 'application/render/restruct';
+import { RasterImage } from 'domain/entities/rasterImage';
+import { RenderOptions } from 'application/render/render.types';
+import { Scale } from 'domain/helpers';
+import { RaphaelElement } from 'raphael';
+import { Vec2 } from 'domain/entities';
+
+export class ReRasterImage extends ReObject {
+  private element?: RaphaelElement;
+
+  static isSelectable(): boolean {
+    return true;
+  }
+
+  constructor(private item: RasterImage) {
+    super('image');
```

Comment:
Please create constant or object with constants to reuse for all such scenarios in future